packages/headless/src/utils/highlight.ts

```diff
@@ -72,3 +101,43 @@ export function highlightString(params: HighlightParams): string {
   }
   return highlighted;
 }
+
+/**
+ * Highlight a suggestion with the given delimiters.
+ * @param suggestion The suggestion to highlight
+ * @param options The object contaning the delimiters used
+ */
+export function getHighlightedSuggestions(
+  suggestion: string,
+  options: SuggestionHighlightingOptions
+) {
+  return suggestion.replace(
+    /\[(.*?)\]|\{(.*?)\}|\((.*?)\)/g,
+    (part, notMatched, matched, corrected) => {
+      if (notMatched) {
+        return suggestionWithDelimiters(notMatched, options.notMatchDelimiters);
+      }
+      if (matched) {
+        return suggestionWithDelimiters(matched, options.exactMatchDelimiters);
+      }
+      if (corrected) {
+        return suggestionWithDelimiters(
+          corrected,
+          options.correctionDelimiters
+        );
+      }
+
+      return part;
+    }
+  );
+}
+
+function suggestionWithDelimiters(
+  suggestion: string,
+  delimiters: OpenCloseDelimiters | undefined
+) {
+  if (delimiters) {
```

Comment:
Hmm, I think this is a better home for the default delimiters. Atm, the defaults are added in the controller, but what would the experience be like for someone using the highlight function directly?

Reply:
Good point, makes sense! 💯 
For a user that uses the highlight function directly, it would add `undefined` as the delimiter and make the string unreadable.

Reply:
The controller, actually doesn't need any default value.